generic3g/OutputInfo.F90

```diff
@@ -223,7 +223,7 @@ function make_ungridded_dim(info, n, rc)
       type(UngriddedDim) :: make_ungridded_dim
```

Comment:
This appears to be a bug.   The `result` is `dims`, but you put the value into the local `make_ungridded_dims` which is the name of the function ...

Reply:
Never mind.   Bad split in GitHub view ...

Reply:
This is from previous PR, but yeah, looks like bug will fix